packages/service-library/src/servicelib/logging_utils.py

```diff
@@ -232,3 +235,9 @@ def log_context(logger: logging.Logger, level: int, msg: str, *args, **kwargs):
     logger.log(level, "Starting " + msg + " ...", *args, **kwargs)
     yield
     logger.log(level, "Finished " + msg, *args, **kwargs)
+
+
+def get_log_record_extra(user_id: int | str | None = None) -> dict | None:
+    if user_id:
```

Comment:
Some tips:

- Expect more ``extra`` in the future
   - Enforce key arguments here (in any case you are using it that way). This way it is easier to keep backwards compatibility when it evolves ...
   - Setup ``extra`` variable
- Defensive programming: assert  to "document" your invariants

```python

def get_log_record_extra(*, user_id: int | str | None = None):
     extra = {}
     if user_id:
         assert int(user_id)>0 # nosec
         extra["log_id"] = f"{user_id}"
     return extra or None
```